server/@types/viewModels/index.d.ts

```diff
@@ -61,13 +61,12 @@ declare module 'viewModels' {
   }
 
   export interface PrisonerSupportNeeds {
-    problemRetrievingData: boolean
-    healthAndSupportNeeds: Array<HealthAndSupportNeeds>
+    lddAssessments: Array<LddAssessment>
+    // TODO - add property for Curious 2 ALN assessments
   }
 
-  export interface HealthAndSupportNeeds {
+  export interface LddAssessment {
```

Comment:
I've renamed this type from `HealthAndSupportNeeds` (which does kinda describe it's intent, but only where there is 1 type of assessment)
Now we are moving to a model of needing to display the LDD and ALN assessments, and we know the data will be different, I've opted to rename this now to `LddAssessment` (with a view that we'll add an `AlnAssessment` or similar in due course)